src/api/tasks.js

```diff
@@ -18,6 +18,16 @@ export const getTasksReq = (pursuanceId) => {
     });
 }
 
+export const deleteTaskReq = (taskGid) => {
+  return postgrest.deleteJSON(`/tasks?gid=eq.${taskGid}`,)
```

Comment:
@ajvb Trailing comma?

Reply:
Oh is that not supposed to be there? Copied it from the above `postgrest.getJSON(`/tasks?pursuance_id=eq.${pursuanceId}&order=created.asc,id.asc`,)` in `getTasksReq`.